code/modules/projectiles/guns/misc/chem_gun.dm

```diff
@@ -45,3 +46,17 @@
 	if(chambered && !chambered.BB)
 		chambered.newshot()
 	last_synth = world.time
+
+/obj/item/gun/chem/bee 
+	name = "bee gun"
+	desc = "An odd piece of BLF tech, which fires bees filled with whatever reagents may be in the gun's reservoir, or venom if there are none."
+	icon_state = "beegun"
+	item_state = "chemgun"
+	fire_sound = 'sound/creatures/bee.ogg'
+	automatic = 1

```

Comment:
```suggestion
	automatic = TRUE

```

Reply:
The original var and all references to it are 1/0 instead of true and false